### PR TITLE
Remember project file after save and add Ps without args

### PR DESCRIPTION
**Pull request by thestr4ng3r**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/rizinbook) with the relevant information (if needed)

**Detailed description**

After saving a project explicitly using `Ps file.rzdb` you can just type `Ps` to save to the same file again.

**Test plan**

run the tests

### CODE REVIEW

File: librz/core/cmd_project.c [hunk 1]
```diff
@@ -3,9 +3,20 @@
 #include <rz_project.h>
 
 RZ_IPI RzCmdStatus rz_project_save_handler(RzCore *core, int argc, const char **argv) {
-	RzProjectErr err = rz_project_save_file (core, argv[1]);
+	const char *file;
+	if (argc == 1) {
+		file = rz_config_get (core->config, "prj.file");
+		if (!file || !*file) {
```
Comment (ret2libc) [inline librz/core/cmd_project.c]:
```suggestion
		if (RZ_STR_ISEMPTY (file)) {
```

File: librz/core/cmd_project.c [hunk 2]
```diff
@@ -3,9 +3,20 @@
...
+			eprintf ("There is no project file associated with the current session yet.\n"
+				"Specify the file explitily as Ps <file.rzdb> or set it manually with e prj.file.\n");
```
Comment (ret2libc) [inline librz/core/cmd_project.c]:
```suggestion
				"Specify the file explitily as Ps <file.rzdb> or set it manually with `e prj.file=<project-path>`.\n");
```

File: librz/core/cmd_project.c [hunk 3]
```diff
@@ -3,9 +3,20 @@
...
+			return RZ_CMD_STATUS_ERROR;
+		}
+	} else { // argc == 2 checked by the shell
+		file = argv[1];
+	}
+	RzProjectErr err = rz_project_save_file (core, file);
 	if (err != RZ_PROJECT_ERR_SUCCESS) {
-		eprintf ("Failed to save project: %s\n", rz_project_err_message (err));
+		eprintf ("Failed to save project: %s to file %s\n", file, rz_project_err_message (err));
```
Comment (ret2libc) [inline librz/core/cmd_project.c]: should you switch the arguments?

**Review by ret2libc — CHANGES_REQUESTED** [review]

File: librz/core/cmd_project.c (hunk 3 above)
Comment (thestr4ng3r) [reply librz/core/cmd_project.c]: The format string was nonsense

**Comment by ITAYC0HEN** [comment]
What about defaulting to save as `path/to/file/filename.rzdb` upon executing `Ps`? We can assume that most likely, people would like to save it with `[filename].rzdb`. IDA assumes it, Binja, and I think even Cutter. It is very probable, and can't hurt (?)

**Comment by thestr4ng3r** [comment]
Yeah could be convenient. It will need a bit of extra logic to for example make sure not to overwrite any other file and to actually determine the binary filename, which is not always possible so should go into another pr.

**Review by ret2libc — APPROVED** [review]

**Pull request merged**